examples/WebFormsButton/main.ipynb

```diff
@@ -0,0 +1,117 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "00d9f193-f748-4f3f-8b44-6541abc461b9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from bspump.http.web.server import *\n",
+    "from bspump.jupyter import *\n",
+    "import asyncio"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "fd948a04-3dfe-4e3c-af8d-fe57cd8762ad",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "new_pipeline(\"WebServerPipeline\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "02bb70c6-2ac8-46aa-af47-8ea59cfdeb95",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"Registering source\")\n",
+    "@register_source\n",
```

Comment:
Since this is a new example please use `auto_pipeine` as that is easier to work with and we want to encourage it's use.